src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAProject.cs

```diff
@@ -154,7 +154,7 @@ TestProject GetTestProject(string name, string target, bool isSdkProject)
             return ret;
         }
 
-        [CoreMSBuildOnlyTheory]
+        [RequiresMSBuildVersionTheory("16.7.0-preview-20228-01")]
```

Comment:
I don't think we should include preview versions here.  Just use `16.7.0` for the value.